### PR TITLE
Remove timeout in async_test for mediacapture-streams tests

### PR DESCRIPTION
**Pull request by qiuzhong**
Remove all the timeout in `async_test` for mediacapture-streams tests.
Affected tests: 12
Before: Pass: 10, Failed: 2
After: Pass: 10, Failed: 2
Related: #11120

### CODE REVIEW

File: mediacapture-streams/MediaStream-add-audio-track.https.html [hunk 1]
```diff
@@ -15,7 +15,7 @@ <h1 class="instructions">Description</h1>
 <script src=/resources/testharness.js></script>
 <script src=/resources/testharnessreport.js></script>
 <script>
-var t = async_test("Tests that adding a track to a MediaStream works as expected", {timeout: 20000}); // longer timeout since requires double user interaction
+var t = async_test("Tests that adding a track to a MediaStream works as expected"); // longer timeout since requires double user interaction
```
Comment (foolip) [inline mediacapture-streams/MediaStream-add-audio-track.https.html]: Can you remove the comment which refers to the deleted code?

File: mediacapture-streams/MediaStreamTrack-end-manual.https.html [hunk 2]
```diff
@@ -17,7 +17,7 @@ <h1 class="instructions">Description</h1>
 <script src=/resources/testharness.js></script>
 <script src=/resources/testharnessreport.js></script>
 <script>
-var t = async_test("Tests that the video MediaStreamTrack objects are properly ended on permission revocation", {timeout: 20000}); // longer timeout since requires user interaction
+var t = async_test("Tests that the video MediaStreamTrack objects are properly ended on permission revocation"); // longer timeout since requires user interaction
```
Comment (foolip) [inline mediacapture-streams/MediaStreamTrack-end-manual.https.html]: remove comment

File: mediacapture-streams/MediaStream-add-audio-track.https.html (hunk 1 above)
Comment (qiuzhong) [reply mediacapture-streams/MediaStream-add-audio-track.https.html]: Oops! Most of the update was done by a script and I didn't notice that unless comment. It's removed now.

File: mediacapture-streams/MediaStreamTrack-end-manual.https.html (hunk 2 above)
Comment (qiuzhong) [reply mediacapture-streams/MediaStreamTrack-end-manual.https.html]: Removed.

**Review by foolip — APPROVED** [review]

**Pull request merged**